quoridor-app/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function App() {
```

Comment:
Use the functional way

Reply:
`const App = () => {}`

Reply:
it is the template of expo, I will change the content